app/messages/_layout.email.html.erb

```diff
@@ -1,62 +1,116 @@
+<!DOCTYPE html>
 <html>
-<body style="background-color: #FFFFFF;">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Canvas Email</title>
```

Comment:
What is this used for?  Can we just get rid of it?  If not, we need to wrap it in t() for translation.